zrcs/zrc-7.md

```diff
@@ -0,0 +1,238 @@
+The localization URI should use one of the following URI schemes defined in this document section URI Schema
+
+| Name          | Type             | Description                                                                       | Required  |
+|---------------| ---------------- | --------------------------------------------------------------------------------- | --------- |
+| `uri`         | `object`         |  A JSON object representing the localization URI pattern.                         | false     |
+| `value`      | `string`         | The URI pattern to fetch localized data from. This URI should contain the substring `{locale}` which will be replaced with the appropriate locale value before sending the request. | false     |
+
+### Default Localization
+
+The language of the default metadata defined in the parent object. This is an optional field. This locale should conform to those defined in the Unicode Common Locale Data Repository.
+
+| Name          | Type             | Description                                                                                 | Required  |
+|---------------| ---------------- | ------------------------------------------------------------------------------------------- | --------- |
+| `default`     | `object`         | A JSON object representing the locale of the default metadata within the parent JSON.       | false     |
+| `value`       | `string`         |The value of the default localization metadata within the parent JSON. This locale should conform to those defined in the Unicode Common Locale Data Repository (<http://cldr.unicode.org/>)        | false     |
+
+### Locale
+
+An array of supported languages. This is an optional field. This locale should conform to those defined in the Unicode Common Locale Data Repository.
+
+| Name          | Type             | Description                                                                            | Required  |
+|---------------| ---------------- | -------------------------------------------------------------------------------------- | --------- |
+| `locale`      | `object`         | A JSON object representing the locale of the default metadata within the parent JSON.  | false     |
+| `value`       | `string`         | The array values of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (<http://cldr.unicode.org/>) | false     |
+
+### B. URI Schema Examples
+
+Developers should use one of the following URI schemes.
+
+When the resource is stored on the web, https should be used.
+```https://example.com/mypict```
+
+When the resource is stored on IPFS, the ```ipfs://``` URI should be used. The IPFS Gateway URI (such as ```https://ipfs.io/ipfs/...```) should not be used.
+```ipfs://QmWS1VAdMD353A6SDk9wNyvkT14kyCiZrNDYAad4w1tKqT```
+
+When the resource is stored on Arweave, https should be used.
```

Comment:
As this is metadata standard. No need to specify which protocol to use to get the metadata.
Remove this line.